__test__/processTestDirectory.test.js

```diff
@@ -24,12 +24,26 @@ describe('processTestDirectory', () => {
       expect(() => validateTestsKeys(invalidRow)).toThrow();
     });
 
+    const validPriority = { ...validRow, assertions: '0:abc 1:abc 2:abc 3:abc' };
+    it('returns row on valid priority', () => {
+      expect(validateTestsKeys(validPriority)).toMatchObject(validPriority);
+    });
+
+    ['0:abc 4:abc', '3:', '4:abc', '5:abc', '6:abc', '7:abc', '8:abc', '9:abc'].forEach(
+      assertion => {
```

Comment:
Just to make sure, these taste cases should cover the possible invalid priority formats almost exhaustively. Let me know if you can think of any other possible invalid cases we should also check for @howard-e !